### ISSUE TITLE
Issue #5 - Peaceful resolution - Improvement

### ISSUE THREAD

**Issue by JulieBousquet** [JulieBousquet/CDD_Step_Peer_Review_2018#5]
https://github.com/JulieBousquet/CDD_Step_Peer_Review_2018/blob/0b0d1cd9545f5ff92ab6de62ded606e0e483e281/CDD2018_DescStat.do#L528

Do the code and graph make sense?
In general, would you have done the same and do you have any suggestion to improve the graph?
I think I am proud of this one but anything can be improved.

**Comment by escandethomas** [comment]
Yeah I agree this one is really nice! Would maybe just put percentage somewhere, but I think it is easy to read
